src/filesystem/impls/appshell/AppshellFileSystem.js

```diff
@@ -112,27 +99,32 @@ define(function (require, exports, module) {
 
     /**
      * Event handler for the Node fileWatcher domain's change event.
+     * event values from chokidar: "change", "ready", "add", "addDir", "unlink", "unlinkDir"
```

Comment:
fixed here: https://github.com/adobe/brackets/pull/12647/commits/c9811ce339f24fd36fc1c8264cfc7126010e0194
also simplified the events so we can use vscode's csharp watcher for windows later